docs/tutorials/notes-mvvm/includes/viewmodel-about.md

```diff
@@ -0,0 +1,91 @@
+---
+author: adegeo
+ms.author: adegeo
+ms.date: 12/21/2022
+ms.topic: include
+no-loc: ["communitytoolkit", "CommunityToolkit.Mvvm", "AllNotes", "Notes", "About"]
+---
+
+Before adding view models to the project, add reference to the MVVM Community Toolkit. This library is available on NuGet, and provides types and systems that help implement the MVVM pattern.
+
+01. In the **Solution Explorer** pane of Visual Studio, right-click on the **Notes** project > **Manage NuGet Packages**.
+01. Select the **Browse** tab.
+01. Search for **communitytoolkit mvvm** and select the `CommunityToolkit.Mvvm` package, which should be the first result.
+01. Make sure that version **8.0.0** is selected.
+01. Next, select **Install** and accept any prompts that are displayed.
+
+    :::image type="content" source="../media/viewmodel-about/nuget.png" alt-text="Searching for the CommunityToolkit.Mvvm package in NuGet.":::
+
+Now you're ready to start updating the project by adding view models.
+
+## Decouple with view models
+
+The view-to-viewmodel relationship relies heavily on the binding system provided by .NET MAUI. The app is already using binding in the views to display a list of notes and to present the text and date of a single note. The app logic is currently provided by the view's code-behind and is directly tied to the view. For example, when a user is editing a note and presses the **Save** button, the `Clicked` event for the button is raised. Then, the code-behind for the event handler saves the note text to a file and navigates to the previous screen.
+
+Having app logic in the code-behind of a view can become an issue when the view changes. For example if the button is replaced with a different input control, or the name of a control is changed, event handlers may become invalid. Regardless of how the view is designed, the purpose of the view is to invoke some sort of app logic and to present information to the user. For this app, the `Save` button is saving the note and then navigating back to the previous screen.
+
+The viewmodel gives the app a specific place to put the app logic regardless of how the UI is designed or how the data is being loaded or saved. The viewmodel is the glue that represents and interacts with the data model on behalf of the view.
+
+The view models are stored in a _ViewModels_ folder.
+
+01. Find the **Solution Explorer** pane of Visual Studio.
+01. Right-click on the **Notes** project and select **Add** > **New Folder**. Name the folder **ViewModels**.
+01. Right-click on the **ViewModels** folder > **Add** > **Class** and name it **AboutViewModel.cs**.
+01. Repeat the previous step and create two more view models:
+    - **NoteViewModel.cs**
```

Comment:
Aren't filenames meant to be italic?

Reply:
I wasn't sure. Because this is interaction with the UI which lists items, they happen to be file names, but it's not like you're working with files or directories via the command line or explorer. Kind of like if you create a new item in a wizard, you don't use _italic_ for the name of the item which is actually going to be a file.

Thoughts on this reasoning? I might also be inconsistent in my usage of this reasoning 😁

Reply:
I get where you're coming from. I'm fine with it being as you've done it.